trinity/sync/common/chain.py

```diff
@@ -14,13 +14,16 @@
 from eth.chains import AsyncChain
 from eth.exceptions import (
     HeaderNotFound,
-    ValidationError as EthValidationError,
+)
+from eth_utils import ValidationError as EthValidationError
```

Comment:
You can drop this rename and just change all of the places where `EthValidationError` is raised to just be a `ValidationError`.